lib/db.ts

```diff
@@ -1,5 +1,8 @@
 import Dexie from 'dexie';
 
+// fake string (uuid) for user id testing
+export const USER_ID = 'c8d6e5f5-9da0-4b0d-8e8d-9da0-4b0d-8e8d';
```

Comment:
Another band-aid until we schlep all this over to a real DB - we'll get the user ID from the session